atlasdb-autobatch/src/test/java/com/palantir/atlasdb/autobatch/TracingEventHandlerTest.java

```diff
@@ -18,20 +18,14 @@
 
 import com.lmax.disruptor.EventHandler;
 import com.palantir.atlasdb.autobatch.DisruptorAutobatcher.DisruptorFuture;
-import com.palantir.tracing.RenderTracingRule;
 import com.palantir.tracing.Tracers;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-/* TODO(boyoruk): Do not forget to upgrade this to JUnit5. */
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
 public class TracingEventHandlerTest {
 
-    @Rule
-    public RenderTracingRule renderTracingRule = new RenderTracingRule();
-
```

Comment:
This has no effect on the test case. It just writes the tracing output to a file. This line is added before to examine what was going on while we are running the test. However, I think we don't need it anymore.